### PR TITLE
AP-4435 fix form grouping accessibility error

### PR DESCRIPTION
**Pull request by RoseSAK**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-4435)

Add an aria-label to the `Name` group on the client details page to make it clear to screen-reader users that the fields `First name` and `Last name` refer to the client name

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- The standards in the [Git Workflow document on Confluence](https://dsdmoj.atlassian.net/wiki/spaces/ATPPB/pages/4602855954/Git+Workflow) should be followed
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

**Review by colinbruce — APPROVED** [review]
LGTM :+1:

File: app/views/shared/forms/_applicant_form.html.erb [hunk 1]
```diff
@@ -14,7 +14,8 @@
       ) do %>
 
     <div class="govuk-!-padding-bottom-2"></div>
```
Comment (naseberry) [inline app/views/shared/forms/_applicant_form.html.erb:16]: CYC: Can we remove these empty "spacer divs" from this view?

Comment (colinbruce) [reply app/views/shared/forms/_applicant_form.html.erb:16]: As @RoseSAK is off until next week, do you want to add a commit and I can approve it?

Comment (naseberry) [reply app/views/shared/forms/_applicant_form.html.erb]: Second commit added @colinbruce and @RoseSAK

**Review by colinbruce — APPROVED** [review]
LGTM :+1:

**Pull request merged**